components/app/project-detail/ProjectSingleRoiStrategy.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <landing-section-container>
+    <h4>ROI <span class="text-primary">strategy</span></h4>
+
+    <p class="small-text text-primary pt-4">{{ $t('project.roi.title') }}</p>
+    <p class="small-text">{{ description }}</p>
+
+    <p class="small-text text-primary pt-4">{{ $t('project.roi.bang') }}</p>
+    <p class="small-text">{{ bang }}</p>
+
+    <p class="small-text text-primary pt-4">{{ $t('project.roi.buck') }}</p>
+    <p class="small-text">{{ buck }}</p>
+
+    <p class="small-text text-primary pt-4">
+      {{ $t('project.roi.success-chance') }}
+    </p>
+    <p class="small-text">{{ chance }}</p>
+
+    <p class="small-text text-primary pt-4">
+      {{ $t('project.roi.calculation') }}
+    </p>
+    <p class="small-text">
+      {{ chance }}
+    </p>
```

Comment:
nitpick (non-blocking): probably a formatting problem? This should be in one line for consistency.